### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by jessesquires**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -94,7 +97,7 @@ This release closes the [2.0.0 milestone](https://github.com/Instagram/IGListKit
 
 - Skip reloading for objects that are not found when calling `-[IGListAdapter reloadObjects:]`. [Ryan Nystrom](https://github.com/rnystrom) (tbd)
 
-- Fixes a crash when a reload is queued for an object that is deleted in the same runloop turn. [Ryan Nystrom](https://github.com/rnystrom) (tbd)
+- Fixes a crash when a reload is queued for an object that is deleted in the same runloop turn. [Ryan Nystrom](https://github.com/rnystrom) [(7c3d499)](https://github.com/Instagram/IGListKit/commit/7c3d4999ebde36ee4666e5aee99716d1ed1fb2d8)
```
Comment (jessesquires) [inline CHANGELOG.md]: @rnystrom is this correct?

File: CHANGELOG.md [hunk 2]
```diff
@@ -94,7 +97,7 @@ This release closes the [2.0.0 milestone](https://github.com/Instagram/IGListKit
 
 - Skip reloading for objects that are not found when calling `-[IGListAdapter reloadObjects:]`. [Ryan Nystrom](https://github.com/rnystrom) (tbd)
 
-- Fixes a crash when a reload is queued for an object that is deleted in the same runloop turn. [Ryan Nystrom](https://github.com/rnystrom) (tbd)
```
Comment (jessesquires) [inline CHANGELOG.md]: @rnystrom what's the sha for this?

File: CHANGELOG.md [hunk 3]
```diff
@@ -94,7 +97,7 @@ This release closes the [2.0.0 milestone](https://github.com/Instagram/IGListKit
 
 - Skip reloading for objects that are not found when calling `-[IGListAdapter reloadObjects:]`. [Ryan Nystrom](https://github.com/rnystrom) (tbd)
```
Comment (jessesquires) [inline CHANGELOG.md]: @rnystrom what's the SHA for this one?

Comment (rnystrom) [inline CHANGELOG.md]: https://github.com/Instagram/IGListKit/commit/ca15e29cf1dadc6c396fe8f14f16c27f6a38519c

**Comment by rnystrom** [comment]
lgtm

**Pull request closed without merging**